core/templates/pages/profile-page/profile-page.component.ts

```diff
@@ -52,28 +52,28 @@ export class ProfilePageComponent {
     value: '',
     helpText: ''
   };
-  usernameIsLong: boolean;
+  usernameIsLong: boolean | undefined;
```

Comment:
Why we allow `undefined`? Ditto below.

Reply:
I believe to handle an error case if any occurs

Reply:
Again, what does it specifically mean that `usernameIsLong` is set to `undefined`? Can we get rid of the `undefined` somehow?